src/gettingstarted/template/check-status.md

```diff
@@ -0,0 +1,52 @@
+# Start with a template
+
+## Build, Deploy, Done!
+
+Once you have configured the template application in the previous step, Platform.sh will build your project for you.
+
+<video controls autoplay loop>
+  <source src="/videos/management-console/check-status.mp4" type="video/mp4">
+</video>
+
+1. **Explore the management console**
+
+   When the build screen has cleared, Platform.sh will return you to the management console. Since you now have a project on your account, a version of this page will be what you see each time you visit the console.
+
+   You will start on the main page for your new project, `My First Project`. From here, you can control the settings of this project and monitor its status.
+
+   In the `Environments` box, click on `Master`.
+
+2. **Check the build status**
+
+   Take a minute to notice some the information available on this page.
+
+   * **Overview**
+
+      In this box the `Master` environment, which is a live environment built from the `master` branch of our application code, will have a status of `Building`.
+
+      Once that status has updated to `Active`, the build is complete and the application has deployed.
+
+   * **Environment Activity**
+
+      In this box, you can see what you have done so far has two initial entries: `My First Project` was created, and the template profile you chose was initialized on the environment.
```

Comment:
Should we use "block" instead of "Box"?  Box feels too informal and "we don't know what to really call this."